server_admin/topics/clients/oidc/proc-using-a-service-account.adoc

```diff
@@ -0,0 +1,65 @@
+[id="proc-using-a-service-account_{context}"]
+[[_service_accounts]]
+
+==== Using a Service Account
+
+Each OIDC client has a built-in _service account_ which allows it to obtain an access token.
+
+.Prerequisites
+* You have selected your client in {project_name}.
+* You are in the `Settings` tab.
+
+.Procedure
+. Set the <<_access-type, Access Type>> of your client is set to `confidential`.
+. Set the `Service Accounts Enabled` switch to ON.
+. Click *Save*
+. Configure your <<_client-credentials, client credentials>>.
+
+In the `Service Account Roles` tab, you can configure the roles available to the service account for your client. These roles must be available in Role Scope Mappings (under the `Scope` tab) of your client as well, unless you have set the `Full Scope Allowed` switch to ON. 
```

Comment:
I would convert this into two steps

. On the Scope tab, verify that you have roles or set `Full Scope Allowed` to ON.
. On the `Service Account Roles` tab, configure the roles available to this service account for your client.